docs/image-variants.md

```diff
@@ -0,0 +1,40 @@
+# Image variants
+
+The Flex API allows requesting custom image variants by accompanying an image
+variant sparse attribute parameter with a corresponding image variant
+definition:
+
+``` js
+sdk.listings.show({
+  id: listingId,
+  include: ["images"],
+  "fields.image": ["variants.my-variant"],
+  "imageVariant.my-variant": "w:640;h:1280;fit:scale"
+}).then(res => {
+  // res.data contains the response data
+});
+```
+
+To clarify what variant options are passed in a given variant definition query
+parameter, the SDK provides a utility function: `sdk.util.imageVariantURLStr`.
+It takes an object as a parameter and returns a string that can be passed as an
+`imageVariant.*` parameter in a Flex API request:
+
+``` js
+sdk.listings.show({
+  id: listingId,
+  include: ["images"],
+  "fields.image": ["variants.my-variant"],
+  "imageVariant.my-variant": sdk.util.imageVariantURLStr({
+    w: 640,
+    h: 1280,
+    fit: 'scale'
+  })
+}).then(res => {
+  // res.data contains the response data
+});
+```
+
+For more information on custom image variants, see [Images resource in the
+Flex API
+reference](https://www.sharetribe.com/api-reference/marketplace.html#image-variants).
```

Comment:
This link doesn't work, will it be updated later?

Reply:
Yes, once the API functionality is released and API docs updated.